programs/intent-transfer/src/bridge/processor/bridge_ntt_tokens.rs

```diff
@@ -473,6 +470,124 @@ pub struct SignedQuote {
     pub signature: [u8; 65],
 }
 
+impl SignedQuote {
```

Comment:
another idea could've been a instroducing a type:
`struct U64BE (u64)`
and manually implementing `BorshSerialize`, `BorshDeserialize` such that it uses big-endian encoding

Reply:
i like this